index.js

```diff
@@ -104,8 +104,6 @@ module.exports = function(S) {
 
       return functions.forEach(function(fun) {
         if( -1 !== fun.getRuntime().getName().indexOf('nodejs') ) {
-          // Override s-function.json defined environment!
-          Object.getPrototypeOf( fun.getRuntime() ).getEnvVars = ()=> BbPromise.resolve( {} );
```

Comment:
It may be made optional, but not removed, because it's really useful to pass local config (different form AWS) via env and not have it overriden by SLS Function.run, e.g. https://github.com/serverless/serverless/blob/master/lib/RuntimeNode.js#L48